GenAI Builder/genaiflows/com/vantiq/genai/TutorialService_runtimeConfig.json

```diff
@@ -1,29 +1,31 @@
 {
   "ars_properties": {
     "graph": {
-      "coordinates": {
+      "scale": 6.368242360929747,
+      "translate": [
+        381,
+        362.5
+      ],
+      "lastZoomRequest": 3,
+      "positions": {
         "Input": {
           "x": 0,
           "y": 0
         }
-      },
-      "lastZoomRequest": 0,
-      "scale": 4.642999253268981,
-      "translate": [
-        570,
-        150.89747573124188
-      ]
+      }
     },
     "groups": {
-      "aicomponents.primitives": true,
-      "aicomponents.resources": true
-    }
+      "aicomponents.resources": true,
+      "aicomponents.primitives": true
+    },
+    "v": 1
   },
   "assembly": {
     "Input": {
-      "children": [],
       "component": "io.vantiq.ai.components.InitialInput",
-      "configuration": {}
+      "configuration": {},
+      "enableAuditing": false,
+      "children": []
```

Comment:
odd that "children" got moved to the end.  This seems to have happened in several of these files.
I thought that export sorted the JSON output to keep VCS diffs to a minimum?

Reply:
No clue -- I just did an export from the UI...